src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -240,9 +258,32 @@ describe.each(['classic', 'visual-refresh', 'visual-refresh-toolbar'] as const)(
         )
       );
 
+      test(
+        'split panel focus toggles between open and close buttons',
+        setupTest(
+          async page => {
+            //Altermatomg between triggers because test-1 trigger hidden in overflow menu on mobile,
+            //security has resize button on desktop
+            const triggerSelector =
+              theme === 'visual-refresh-toolbar'
+                ? wrapper.find(`button[data-testid="awsui-app-layout-trigger-slide-panel"]`).toSelector()
+                : wrapper.findSplitPanel().findOpenButton().toSelector();
+            await page.click(triggerSelector);
+            await page.isFocused(wrapper.findSplitPanel().findSlider().toSelector());
+            await page.keys(['Tab', 'Tab']);
+            await page.isFocused(wrapper.findSplitPanel().findCloseButton().toSelector());
+            await page.keys('Enter');
+            await expect(page.isFocused(triggerSelector)).resolves.toBe(true);
+            await page.keys('Enter');
+            await page.isFocused(wrapper.findSplitPanel().findSlider().toSelector());
```

Comment:
same as above. btw, do you find this statement valuable? this test has already checked focus shifting from trigger button to close and backwards